### PR TITLE
Readded Header Data information to models

### PR DESCRIPTION
**Pull request by OsiJr**

### CODE REVIEW

File: docs/TranslationGuide.md [hunk 1]
```diff
@@ -713,3 +723,27 @@ Modules are currently:
...
+- `attributeTypeInt32`: label when the attribute is a 32-bit signed integer
+- `attributeTypeInt64`: label when the attribute is a 64-bit signed integer
+- `attributeTypeFloat32`: label when the attribute is a 32-bit floating point value
+- `attributeTypeFloat64`: label when the attribute is a 64-bit floating point value
+- `attributeTypeColour`: label when the attribute is the special colour format
+- `attributeTypeNormal`: label when the attribute is a packed normal
+- `attributeTypeVec3F32`: label when the attribute is a 3x 32-bit floating point values (3D Vector)
+- `attributeTypeUnknown`: label when the attribute is an unknown format
+
+- `attributeBlending": Label when showing the attribute blending mode
```
Comment (samsinsane) [inline docs/TranslationGuide.md]: You've closed with a quote here (and the three below) instead of a grave.

File: docs/TranslationGuide.md [hunk 2]
```diff
@@ -618,6 +620,14 @@ Modules are currently:
 - `sceneModelPosition`: Label used for the model position
 - `sceneModelRotation`: Label used for the model rotation
 - `sceneModelScale`: Label used for the model scale
+- `sceneModelMeterScale`: Label for metadata on meter scale for this model (how many units per metric meter)
+  - {0} the actual scale amount
+- `sceneModelLODLayers`: Label for metadata on number of LOD layers for this model
+  - {0} the number of LOD layers in this model
+- `sceneModelResolution`: Label for metadata on resolution at time of conversion for this model
+  - {0} the resolution of the model
+- `sceneModelAttributes: Label for metadata on number of attributes for this model
```
Comment (DamianMadden) [inline docs/TranslationGuide.md]: No grave closes here

Comment (samsinsane) [reply docs/TranslationGuide.md]: I was wondering why all of the formatting died, nice find!

File: docs/TranslationGuide.md [hunk 3]
```diff
@@ -618,6 +620,14 @@ Modules are currently:
 - `sceneModelPosition`: Label used for the model position
 - `sceneModelRotation`: Label used for the model rotation
 - `sceneModelScale`: Label used for the model scale
+- `sceneModelMeterScale`: Label for metadata on meter scale for this model (how many units per metric meter)
```
Comment (DamianMadden) [inline docs/TranslationGuide.md]: Metre*

File: src/scene/vcModel.cpp [hunk 4]
```diff
@@ -285,6 +286,92 @@ void vcModel::HandleImGui(vcState *pProgramState, size_t * /*pItemID*/)
     vdkProjectNode_SetMetadataDouble(m_pNode, "transform.scale", scale.x);
   }
 
+  // Show MetaData Info
+  {
+    char buffer[128];
+
+    ImGui::TextUnformatted(vcStringFormat(buffer, udLengthOf(buffer), vcString::Get("sceneModelMeterScale"), udTempStr("%.4f", m_pointCloudHeader.unitMeterScale)));
```
Comment (DamianMadden) [inline src/scene/vcModel.cpp]: unitMetreScale*

File: src/scene/vcModel.cpp [hunk 5]
```diff
@@ -285,6 +286,92 @@ void vcModel::HandleImGui(vcState *pProgramState, size_t * /*pItemID*/)
...
+        case vdkAttributeTypeInfo_normal32:
+          vcStringFormat(buffer, udLengthOf(buffer), vcString::Get("attributeType"), vcString::Get("attributeTypeNormal"));
+          break;
+        case vdkAttributeTypeInfo_vec3f32:
+          vcStringFormat(buffer, udLengthOf(buffer), vcString::Get("attributeType"), vcString::Get("attributeTypeVec3F32"));
+          break;
+        default:
+          vcStringFormat(buffer, udLengthOf(buffer), vcString::Get("attributeType"), vcString::Get("attributeTypeUnknown"));
+          break;
+        }
```
Comment (DamianMadden) [inline src/scene/vcModel.cpp]: I'd prefer a map where these enum indices map directly to the fetched strings

Comment (OsiJr) [reply src/scene/vcModel.cpp]: That would leave us in a state where we need to alloc and cleanup the map and ultimately would be the same amount of code and complexity.

This is untidy but its explicit in what it's doing.

Comment (DamianMadden) [reply src/scene/vcModel.cpp]: in this instance I mean map as in an array where indices map to the vcString entry, no need to dynamically alloc or clean up and presumbly more efficient in that this cascading switch structure is replace by a calculation then jump

Comment (samsinsane) [reply src/scene/vcModel.cpp]: Just to clarify, are you suggesting something like:
```c++
const char *pAttrStringMap[vdkAttributeTypeInfo_Count] = {
  vcString::Get("attributeTypeUint8"),
  ...
};
vcStringFormat(buffer, udLengthOf(buffer), vcString:Get("attributeType"), pAttrStringMap[m_pointCloudHeader.attributes.pDescriptors[i].typeInfo]);
```
I don't think those VDK values are sequential from 0, so an array wouldn't really work?

Comment (OsiJr) [reply src/scene/vcModel.cpp]: Sorry, I misunderstood; @samsinsane is right, these are not-sequential. The values are encoded data about the type.

The values (encoded) are:
```
    vdkAttributeTypeInfo_uint8 = 1,
    vdkAttributeTypeInfo_uint16 = 2,
    vdkAttributeTypeInfo_uint32 = 4,
    vdkAttributeTypeInfo_uint64 = 8,
    vdkAttributeTypeInfo_int8 = 65537,
    vdkAttributeTypeInfo_int16 = 65538,
    vdkAttributeTypeInfo_int32 = 65540,
    vdkAttributeTypeInfo_int64 = 65544,
    vdkAttributeTypeInfo_float32 = 196612,
    vdkAttributeTypeInfo_float64 = 196616,
    vdkAttributeTypeInfo_color32 = 262148,
    vdkAttributeTypeInfo_normal32 = 524292,
    vdkAttributeTypeInfo_vec3f32 = 197388
```

The `vdkAttributes` header has more information on these types and how to decode them

**Pull request merged**